src/Avalonia.Controls/NativeMenuItemSeparator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Avalonia.Controls
+{
+
+    [Obsolete("This class exists to maintain backwards compatiblity with existing code. Use NativeMenuItemSeparator instead")]
+    public class NativeMenuItemSeperator : NativeMenuItemSeparator 
```

Comment:
Btw, I think we can log warning, if this class' ctor was called.

Reply:
I wonder if there is value in that, many people don't have logging enabled.
Useful in Avalonia codebase but only because we have so many warnings that we can miss this.